integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -82,130 +84,139 @@ public class MultiTenantIT {
     private static final String MY_VALUE = "my-value";
     private TestInfo testInfo;
     private KeytoolCertificateGenerator certificateGenerator;
+    private Path clientTrustStore;
 
     @BeforeEach
     public void beforeEach(TestInfo testInfo) throws Exception {
         this.testInfo = testInfo;
         this.certificateGenerator = new KeytoolCertificateGenerator();
         this.certificateGenerator.generateSelfSignedCertificateEntry("test@redhat.com", "*.multitenant.kafka", "KI", "RedHat", null, null, "US");
+        Path certsDirectory = Files.createTempDirectory("kafkaClient");
```

Comment:
suggest using junits TempDir annotation on a field like:
```
@TempDir
private Path certsDirectory;
```
It'll get a fresh temp dir per test that is recursively deleted at the end so you don't need to set up your `deleteOnExit`s

Reply:
thanks! I didn't know that annotation